x/stake/types/validator.go

```diff
@@ -5,15 +5,16 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/crypto"
 	"github.com/tendermint/tendermint/crypto/tmhash"
 	tmtypes "github.com/tendermint/tendermint/types"
-
-	"github.com/cosmos/cosmos-sdk/codec"
-	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const MockSideChainIDForBeaconChain = "beacon_chain"
```

Comment:
Not sure which is better `bc` or `beacon_chain`? We can align to binance smart chain.

Reply:
Its usage is to hash the string and xor with the operator address to generate DistributionAddr.
Length doesn't matter here. IMO, `beacon_chain` is more clear.